src/Application/Cli/PreviewCommand.php

```diff
@@ -225,7 +225,7 @@ private function startLivereload(
 
     private function isSupported(): bool
     {
-        if (version_compare(phpversion(), '5.4.0', '<')) {
```

Comment:
That `isSupported()` is for checking that PHP >= 5.4 because that's the version they added the built-in webserver.

We could completely remove that check now instead of upgrading the error message.

Reply:
Done